src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -929,6 +929,7 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode, callable $e
                     ->fixXmlConfig('fallback')
                     ->fixXmlConfig('path')
                     ->fixXmlConfig('provider')
+                    ->fixXmlConfig('default_parameter')
```

Comment:
or just "parameter"?

Reply:
Or "global_parameters"?

Reply:
or just "global" like in twig? :D

Reply:
So let's go for the following naming ?

`globals` in configuration
```yaml
framework:
    translator:
        globals:
            '{version}': '1.2.3'
            '{project_dir}': '%kernel.project_dir%'
```

And `default_parameters` for the service
```php
$container->register('translator.default_parameters', DefaultParametersTranslator::class)
```

Reply:
yes please :)

Reply:
Done :slightly_smiling_face: